Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -12823,8 +12836,322 @@ def pan_os_delete_application_group_command(args):
         readable_output=f'application-group {application_group_name} was deleted successfully.'
     )
 
+""" Fetch Incidents """
+
+
+def get_query_job_id_request(log_type: str, query: str, max_fetch: int) -> str:
+    """get the Job ID linked to a particular query.
+
+    Args:
+        log_type (str): query log type
+        query (str): query for the fetch
+        max_fetch (int): maximum number of entries to fetch
+
+    Returns:
+        job_id (str): returns the Job ID associated with the given query
+    """
+    params = assign_params(key=API_KEY, type='log', log_type=log_type.lower(), query=query, nlogs=max_fetch, skip=0)
```

Comment:
I think its equal to "page" or "offset" argument, meaning skipping number of results returned from the HTTP request. the request actually works without it so its optional, but because we do not support it as a parameter maybe we should remove it?

Reply:
depends on the if the api supports 0 by default? if yes, you can remove it, if not then keep it

Reply:
The default is 0, i'm removing it.